SDLnetsys.h

```diff
@@ -49,18 +49,24 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <netinet/in.h>
-#ifndef __BEOS__
+#if !defined(__BEOS__) || defined(__PSP__)
```

Comment:
This does not seem to be needed: Already was always including it for non-beos.

Reply:
You're right.